dynamodb-enhanced/runtime/src/main/java/io/quarkus/amazon/dynamodb/enhanced/runtime/NamedDynamoDbTable.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.amazon.dynamodb.enhanced.runtime;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.inject.Qualifier;
+
+/**
+ * Specification of DynamoDb table to be injected.
+ */
+@Target({ FIELD, METHOD, PARAMETER })
```

Comment:
Is this qualifier applicable to method and parameter? Do we handle this in DynamodbEnhancedDbTableProcessor or anywhere else? If so, sorry, I'm still learning to write quarkus extensions, could be possible that I missed something ;-)

Reply:
@knuspertante Thanks. you are right, injection point are discovered in `DynamodbEnhancedDbTableProcessor` and it only processes fields